### DISCUSSION TITLE
I get Rejected when applying for Student benefits because I apparently am not near the school.

### DISCUSSION THREAD

**Discussion by PluT0nic** [community/community#48350] [GitHub Education]
Hi, so I'm a bit of a beginner when it comes to coding websites and so I heard bootstrap studio can help with doing the CSS part of a website for you. I then heard of the student benefits and immediately signed up using all the required information. To my surprise, it said I wasn't located near the school and that is why they rejected me. What's funny is I actually requested it while I was in school and at home and got rejected both time and I can't figure out why it's doing this. Maybe there is something preventing it from reading my location but I know my school info and everything is correct. If anyone has a solution to this problem, I'd be grateful if you could help me out!

**Comment by jge162** [comment]
What email are you using? to sign up for the program?

**Reply by PluT0nic** [reply]
I had to use my personal one since my school blocks any emails outside of the school.

**Comment by samus-aran** [comment]
Hi @PluT0nic 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here are a few tips to help when applying:

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are using a VPN, please disable it when you reapply. If you are not using a VPN, try reapplying when connected to the internet from a different network.

Hope this helps. Let us know how you get on 👍
